deployments/docker-compose/docker-compose.yaml

```diff
@@ -7,12 +7,16 @@ services:
       - 18080:18080
     links:
       - mongodb
+      - minio
     environment:
       - WEB_HOST=${WEB_HOST} # Should be
       - SERVER_SECRET=${SERVER_SECRET}
       - MONGODB_URI=mongodb://mongodb:27017
       - ACCOUNTS_URI=https://${WEB_HOST}/auth
       - CLIENT_URI=wss://${WEB_HOST}:18080
+      - S3_URI='http://127.0.0.1:9000',
```

Comment:
We need to use ssl, or we need to have a mode to passthrough data via our ssl attachment entry point.